[ringbuffer] RingbufferAddAllReadManyStressTest.whenShortTTLAndBigBuffer

```
No error should have been thrown, but ConsumeThread-1 completed error expected null, but was:
<java.util.concurrent.ExecutionException: com.hazelcast.ringbuffer.StaleSequenceException: 
sequence:1123660 is too small. The current headSequence is:1123668>
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.maintenance-nightly/509/com.hazelcast$hazelcast/testReport/com.hazelcast.ringbuffer.impl/RingbufferAddAllReadManyStressTest/whenShortTTLAndBigBuffer/

```
java.lang.AssertionError: No error should have been thrown, but ConsumeThread-2 completed error expected null, but was:<java.util.concurrent.ExecutionException: com.hazelcast.ringbuffer.StaleSequenceException: sequence:76334 is too small. The current headSequence is:76357>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:755)
	at org.junit.Assert.assertNull(Assert.java:737)
	at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:60)
	at com.hazelcast.ringbuffer.impl.RingbufferAddAllReadManyStressTest.test(RingbufferAddAllReadManyStressTest.java:87)
	at com.hazelcast.ringbuffer.impl.RingbufferAddAllReadManyStressTest.whenShortTTLAndBigBuffer(RingbufferAddAllReadManyStressTest.java:62)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast%20Maintenance/job/Hazelcast-3.maintenance-nightly/com.hazelcast$hazelcast/691/testReport/junit/com.hazelcast.ringbuffer.impl/RingbufferAddAllReadManyStressTest/whenShortTTLAndBigBuffer/

```
java.lang.AssertionError: No error should have been thrown, but ConsumeThread-2 completed error expected null, but was:<java.util.concurrent.ExecutionException: com.hazelcast.ringbuffer.StaleSequenceException: sequence:82927 is too small. The current headSequence is:82989>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:755)
	at org.junit.Assert.assertNull(Assert.java:737)
	at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:60)
	at com.hazelcast.ringbuffer.impl.RingbufferAddAllReadManyStressTest.test(RingbufferAddAllReadManyStressTest.java:87)
	at com.hazelcast.ringbuffer.impl.RingbufferAddAllReadManyStressTest.whenShortTTLAndBigBuffer(RingbufferAddAllReadManyStressTest.java:62)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast%20Maintenance/job/Hazelcast-3.maintenance-nightly/com.hazelcast$hazelcast/696/testReport/junit/com.hazelcast.ringbuffer.impl/RingbufferAddAllReadManyStressTest/whenShortTTLAndBigBuffer/

```
java.lang.AssertionError: No error should have been thrown, but ConsumeThread-1 completed error expected null, but was:<java.util.concurrent.ExecutionException: com.hazelcast.ringbuffer.StaleSequenceException: sequence:82361 is too small. The current headSequence is:82386>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:755)
	at org.junit.Assert.assertNull(Assert.java:737)
	at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:60)
	at com.hazelcast.ringbuffer.impl.RingbufferAddAllReadManyStressTest.test(RingbufferAddAllReadManyStressTest.java:86)
	at com.hazelcast.ringbuffer.impl.RingbufferAddAllReadManyStressTest.whenShortTTLAndBigBuffer(RingbufferAddAllReadManyStressTest.java:62)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast%20Maintenance/job/Hazelcast-3.maintenance-nightly/com.hazelcast$hazelcast/704/testReport/junit/com.hazelcast.ringbuffer.impl/RingbufferAddAllReadManyStressTest/whenShortTTLAndBigBuffer/

--- a/hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferAddAllReadManyStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferAddAllReadManyStressTest.java
@@ -42,7 +42,7 @@ public class RingbufferAddAllReadManyStressTest extends HazelcastTestSupport {
     public void whenTTLEnabled() throws Exception {
         RingbufferConfig ringbufferConfig = new RingbufferConfig("foo")
                 .setCapacity(200 * 1000)
-                .setTimeToLiveSeconds(1);
+                .setTimeToLiveSeconds(2);
         test(ringbufferConfig);
     }
 
@@ -50,7 +50,7 @@ public class RingbufferAddAllReadManyStressTest extends HazelcastTestSupport {
     public void whenLongTTLAndSmallBuffer() throws Exception {
         RingbufferConfig ringbufferConfig = new RingbufferConfig("foo")
                 .setCapacity(1000)
-                .setTimeToLiveSeconds(1);
+                .setTimeToLiveSeconds(30);
         test(ringbufferConfig);
     }
 
@@ -58,7 +58,7 @@ public class RingbufferAddAllReadManyStressTest extends HazelcastTestSupport {
     public void whenShortTTLAndBigBuffer() throws Exception {
         RingbufferConfig ringbufferConfig = new RingbufferConfig("foo")
                 .setCapacity(20 * 1000 * 1000)
-                .setTimeToLiveSeconds(1);
+                .setTimeToLiveSeconds(2);
         test(ringbufferConfig);
     }
 

